scripts/script.py

```diff
@@ -36,8 +26,12 @@
 )
 
 # Figure out how to initialize and fit models using the training data
-incident_model = iup.LinearIncidentUptakeModel().fit(incident_train_data).predict(
-    config["timeframe"]["start"],
-    config["timeframe"]["end"],
-    config["timeframe"]["interval"],
+incident_model = (
+    iup.LinearIncidentUptakeModel()
+    .fit(incident_train_data)
+    .predict(
```

Comment:
if the keywords here are `start`, `end`, `interval`, then you could do `.predict(**config['timeframe'])`

sometimes that's really slick to do `f(**my_dict)`, but other times it is better to be explicit with `f(arg1=my_dict['arg1'], arg2=my_dict['arg2'])`